types/inferschematype.d.ts

```diff
@@ -219,27 +219,31 @@ type IsSchemaTypeFromBuiltinClass<T> = T extends (typeof String)
             ? true
             : T extends (typeof Schema.Types.Decimal128)
               ? true
```

Comment:
it seems like there are a lot of unintended changes in this file?

Reply:
Removed them, no longer necessary since mongoose.Type.Int32 doesn't exist.